static/js/typing_events.js

```diff
@@ -52,7 +53,9 @@ function get_users_typing_for_narrow() {
 }
 
 export function render_notifications_for_narrow() {
-    const user_ids = get_users_typing_for_narrow();
+    let user_ids = get_users_typing_for_narrow();
+    user_ids = user_ids.filter((id) => !muting.is_user_muted(id));
```

Comment:
I think it'd be better to push this down into the `typing_data.js` logic -- we can rename `get_all_typists` to `get_unmuted_typists` and do the check there.

Reply:
Done!